apps/civicsignalblog/src/components/PageHeader/PageHeader.js

```diff
@@ -31,9 +32,13 @@ const PageHeader = React.forwardRef(function PageHeader(props, ref) {
         >
           {title}
         </RichTypography>
-        <RichTypography component="h2" variant="h2">
-          {subtitle}
-        </RichTypography>
+        <RichText
+          typographyProps={{
```

Comment:
Not sure how we missed this but lets add an issue to track this, `typographyProps` should be `TypographyProps`